stagelib/git.py

```diff
@@ -17,3 +17,10 @@ def get_branch():
 
 def get_hash():
     return _run_cmd(['git', 'rev-parse', '--verify', 'HEAD'])
+
+
+def file_is_dirty(file_path):
+    file_status_msg = _run_cmd(['git', 'status', str(file_path)])
+    if 'nothing to commit' in file_status_msg:
```

Comment:
Be careful, you are making an assumption about the locale. We are not all using en_US ;)
Maybe run_cmd() should enforce the locale to get a constant output.

Reply:
Good catch. I'll see if there is a better way to get file dirty status or enforce a locale.

Reply:
Fixed in latest `push -f`